src/main/java/Actions/Deadline.java

```diff
@@ -0,0 +1,24 @@
+package Actions;
+
+import java.util.ArrayList;
+import Tasks.Task;
+import Exceptions.*;
+
+public abstract class Deadline extends Template {
+    protected static String trigger = "deadline";
+
+    protected static Boolean isTrigger(String input, String trigger) {
+        return input.toLowerCase().replaceAll(" ", "").replaceAll("[0-9]", "").equals(trigger);
+    }
+
+    public static Boolean run(String input, ArrayList<Task> list) throws ArrayIndexOutOfBoundsException, ValueNotFound, InvalidInput {
+        if (isTrigger(input, trigger)) throw new ValueNotFound("You need to provide a value for deadline");
+        if (!isTriggerPrefix(input, trigger)) return false;
```

Comment:
Should the conditional be put on a single line and not wrapped by curly brackets? Even if its just a single statement. I noticed the same issue in several other places too.